framework/base/src/macros.rs

```diff
@@ -202,6 +202,11 @@ macro_rules! only_owner {
     };
 }
 
+#[macro_export]
+macro_rules! allow_multiple_var_args {
```

Comment:
No need for this.

Our proc-macro gets rid of annotations it processes. This also helps us easier detect cases where attributes are not properly processed, or we missed something.

If it doesn't work without it, it means we missed something.